app/scripts/modules/core/src/pipeline/config/stages/googleCloudBuild/GoogleCloudBuildStageConfig.tsx

```diff
@@ -0,0 +1,69 @@
+import * as React from 'react';
+import Select, { Option } from 'react-select';
+
+import { IgorService, IStageConfigProps, StageConfigField, YamlEditor, yamlDocumentsToString } from '@spinnaker/core';
+
+export interface IGoogleCloudBuildStageConfigState {
+  googleCloudBuildAccounts: string[];
+  rawBuildDefinitionYaml: string;
+}
+
+export class GoogleCloudBuildStageConfig extends React.Component<IStageConfigProps, IGoogleCloudBuildStageConfigState> {
+  public constructor(props: IStageConfigProps) {
+    super(props);
+    this.state = {
+      googleCloudBuildAccounts: [],
+      rawBuildDefinitionYaml: props.stage.buildDefinition ? yamlDocumentsToString([props.stage.buildDefinition]) : '',
+    };
+  }
+
+  public componentDidMount = (): void => {
+    if (this.props.stage.isNew) {
+      this.props.updateStageField({ application: this.props.application.name });
+    }
+    this.fetchGoogleCloudBuildAccounts();
```

Comment:
What are your thoughts on catching this when it errors out? Do we have an existing convention for alerting the user / logging? When it 404's it throws an uncaught exception (no existing accounts).

Reply:
No convention that I know of; unfortunately looks like many catches are nonexistent or noops. I feel like this would be a great UI SIG discussion though, with the goal of landing on one right way to handle

Reply:
I this particular case, the endpoint will return an empty list (rather than 404) when there are no existing accounts, so that's not an error mode to worry about.  But agree with the general sentiment of finding a good way to handle general error cases.

Reply:
Added to UI SIG agenda for 4/8 :tada:

Reply:
@ezimanyi I'm seeing the endpoint return a 404 locally. This is without setting up any GCB accounts:

![image](https://user-images.githubusercontent.com/2623242/55238206-4d354c80-520a-11e9-9d7c-7cd71832479a.png)

I agree that error handling conventions could be good to cover at the SIG 👍

Reply:
It will return a 404 if you don't have the `gcb` feature flag enabled at all in `igor` (as those endpoints don't exist at all)...so I guess this is something that a user could see if they haven't configured properly and might make sense to improve the error message around.